transactions/update-contract/2025/may-13-migration-contract-upgrade/Migration-contract-update.cdc

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
nit: maybe remove these 2 empty lines.

Reply:
I use what was merged here: https://github.com/onflow/flow-go/pull/7344/files#diff-c6a4158cb65be808d04ca9c9511e82585c373d89250cb7f60b3dcf26c073fd73, but happy to remove them :)